ml_data_management_code.R

```diff
@@ -4,16 +4,266 @@
 ##
```

Comment:
I would consider making this a markdown file or notebook. Your comments are trying to communicate a more substantial message than what is typically communicated in a comment. However, I think an R script is fine, if you prefer working with that. This is also a change we can make later.

Reply:
I like the idea of changing to a notebook / markdown as well. You make a fair point.

Reply:
I like this idea too!